### ISSUE TITLE
YSFGateway is not reverting back to default reflector !!!!!!

### ISSUE THREAD

**Issue by iw9grl** [g4klx/YSFClients#99]
Could someone verify please ?

tnx
73

**Comment by ve1aic** [comment]
Yes, despite 'Revert=1' and 'InactivityTimeout=10' being set in YSFGateway.ini this function no longer works on current YSFGateway.

**Comment by iw9grl** [comment]
Hoping a fix!

**Comment by m0vse** [comment]
This is still an issue, it looks like m_inactivityTimer is never getting started. I have verified this by manually starting it if not running and the current reflector is not the default one. The reflector is then correctly disconnected after timeout. I am still testing it as there are parts of the code that i'm not sure what their function is so there may be a better fix. If not, I will create a pull request.

**Comment by amvidalrc** [comment]
Any solution or workaround for this? I need my DMO to roll back automatically to the default TG...

**Comment by iw9grl** [comment]
I'm waiting from 16 Feb !!! :(

**Comment by g4klx** [comment]
That is disgusting!
I do this project for fun in my spare time for no money. You have no right to demand anything from me.


Sent from Yahoo Mail for iPhone


On Thursday, May 24, 2018, 15:29, iw9grl <notifications@github.com> wrote:


I'm waiting from 16 Feb !!! :(

**Comment by m0vse** [comment]
I quite agree Jonathan, I have put a workaround on my repeater which seems to work but it is a bit of a hack which is forcing m_inactivityTimer to start (if it isn't already) which suggests that something is stopping it being started in the first place? 

You provide all of the tools for people to fix things themselves, if they aren't willing and/or able to do this then they have no right to complain when it isn't at the top of your list.

**Comment by iw9grl** [comment]
Jonathan I have no claim and never allow myself to claim anything from anyone, in fact I have always respected the work of others and obviously yours, that you can say that what I wrote disgusts you at least seems exaggerated, what should disgust you  what other people do by speculating on your project,
That said, I wish you a good day

**Comment by m0vse** [comment]
Last nights commit seems to have fixed this issue as I have replaced my "hack" and the repeater successfully reverted to the default reflector. Thanks Jonathan!

**Comment by iw9grl** [comment]
With last commit I have a problem to load the YSF Reflector list, after start it I have Loaded 0 YSF Reflector

**Comment by m0vse** [comment]
I had a similar issue which was related to a problem with curl, check that your YSFHosts.txt exists and contains the reflectors.

**Comment by iw9grl** [comment]
Now the revert work well, I want to thank you Jonathan .... but instead a reconnection of the default YSFReflector,  it send a disconnection !
